commands/trigger.go

```diff
@@ -594,10 +579,42 @@ func (t *Trigger) Update(Client *whisk.Client, args []string) error {
 			}
 		}
 
+		//if there is no feed attached to this trigger
+		if len(fullFeedName) < 1 {
+			//but user indicate feed parameter change, we issue error message.
+			if len(Flags.trigger.feedParam) > 0 {
+				whisk.Debug(whisk.DbgError, "trigger %+v is found but it does not contain a feed. \n", trigger)
+				errStr := wski18n.T("triggerDoesNotContainFeed", map[string]interface{}{"name": qualifiedName.GetEntityName()})
```

Comment:
Please look at using a constant in https://github.com/apache/openwhisk-cli/blob/master/commands/messages.go instead of a string `triggerDoesNotContainFeed` which requires a matching entry in the localization file.

Reply:
`triggerDoesNotContainFeed` represents a very specific message that tells users they've run into a specific situation that will results to error. This happens when they tries to update feed params when the trigger contains no feed. I cannot find any constant in messages.go file that represents this. There is only one constant that's relevant, which is `FEED_CONFIGURATION_FAILURE`, but this is too broad and general error message.

Reply:
You are correct. I do not need to add another string in the resource file, so I've changed it.